documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p><code>before you begin experimenting with neural networks. </code></p><p> </p><p>Try not to down-play DFs. They are a production-ready option that provide a number of benefits over DNN-style architectures.</p><p> </p><p>Perhaps here just write "...will often outperform neural networks."</p><p> </p><p>Also, I think the first paragraph under "random forest" repeats much of the introduction paragraph. Can you remove the "Random Forest" heading and flow this section on from the introduction? I think we can keep the intro short &amp; clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>